README.md

```diff
@@ -166,6 +166,9 @@ the one you are trying to build.
 -k, --keep
 if using docker, keep the container after the build.
 
+--make-exploded
+creates an exploded image (useful for codesigning jmods). Use --assemble-exploded-image once you have signed the jmods to complete the packaging steps.
```

Comment:
So it's 'compulsory' to use both together?  I'd make that clear

Reply:
it's not compulsory, most people would want to run the latter for packing